### PR TITLE
initial release note draft for 6.0

### PR DESCRIPTION
**Pull request by zkat**

### CODE REVIEW

**Review by JonDouglas — COMMENTED** [review]
Contents look great but can definitely use a PM pass for all the features introduced. I can help with screenshots, blurbs, and more to help make the blog consistent.

File: docs/release-notes/NuGet-6.0.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+ms.topic: conceptual
+---
+
+# NuGet 6.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
```
Comment (JonDouglas) [inline docs/release-notes/NuGet-6.0.md:16]: NuGet - 6.0
Visual Studio 2019 version 16.11.4
Visual Studio 2022 version 17.0
.NET 5.0.402
.NET 6.0.0-rc.2


I believe these are right? May need to dig a little further though.

**Comment by nkolev92** [comment]
My docs are gonna reference the doc being added here: https://github.com/NuGet/docs.microsoft.com-nuget/pull/2546. 

I haven't had a chance to review this yet, I'll do it next week.

File: docs/TOC.md [hunk 2]
```diff
@@ -292,6 +292,9 @@
 ### [NuGet.org policies](nuget-org/policies/data-requests.md)
 ## Release notes
 ### [Known Issues](release-notes/known-issues.md)
+
```
Comment (nkolev92) [inline docs/TOC.md:295]: style: There are no spaces anywhere in the TOC

File: docs/release-notes/NuGet-6.0.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+> [!NOTE]
+> Visual Studio 17.0, MSBuild 17.0, and .NET 6.0 require NuGet.exe 6.0 or later.
+
+## Summary: What's New in 6.0
+
+🎉 **This is the first release to offer full authoring and restoring support for NuGet packages targeting .NET 6.0** 🎉
+
+* Add  hook for excluding certain build output extensions from the nuget package  - [#10690](https://github.com/NuGet/Home/issues/10690)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:27]: We tend to order things by priority/importance here.

Comment (nkolev92) [reply docs/release-notes/NuGet-6.0.md:27]: While this is a feature, I'd argue it's a relatively small one, so I'd put it toward the lower end of this list.

File: docs/release-notes/NuGet-6.0.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+
+* Implement new precedence for xamarin TFMs when using net6.0+ - [#10717](https://github.com/NuGet/Home/issues/10717)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:29]: Idk, if this on it's own is super relevant given that we have a big shout to the fact that net6.0 frameworks are supported.

File: docs/release-notes/NuGet-6.0.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+* Implement maccatalyst warning when "falling back" to xamarin.ios for net6.0+ projects - [#10718](https://github.com/NuGet/Home/issues/10718)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:31]: Same as above, I'd probably move this to a generic list as it duplicates the first part.

File: docs/release-notes/NuGet-6.0.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+
+* Add package namespaces support as part of the configuration - add a reading capability only - [#10725](https://github.com/NuGet/Home/issues/10725)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:33]: This is a bug under a larger feature. I think we should indicate restore support for package source mapping, and not bother with all these issues.

File: docs/release-notes/NuGet-6.0.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+
+* Package Namespaces: Account for package namespaces filtering in package downloading in PackageReference restore - [#10732](https://github.com/NuGet/Home/issues/10732)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:35]: Same as above, call out the larger feature only.

File: docs/release-notes/NuGet-6.0.md [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+
+* Add namespaces filtering support in packages.config restore in nuget.exe and msbuild /t:restore - [#10737](https://github.com/NuGet/Home/issues/10737)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:37]: Same as above, call out the larger feature only.

File: docs/release-notes/NuGet-6.0.md [hunk 9]
```diff
@@ -0,0 +1,280 @@
...
+
+* Add a contract for identifying a restore source, to help nuget better batch restores/branch switches.  - [#10807](https://github.com/NuGet/Home/issues/10807)
+
+* Add support for net6.0-tizen and net6.0-android - [#10819](https://github.com/NuGet/Home/issues/10819)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:41]: Duplicates the first line.

File: docs/release-notes/NuGet-6.0.md [hunk 10]
```diff
@@ -0,0 +1,280 @@
...
+
+* Add namespaces filtering support for packages.config restore in Visual Studio - [#10823](https://github.com/NuGet/Home/issues/10823)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:43]: Same as above, should be called out as a general feature.

File: docs/release-notes/NuGet-6.0.md [hunk 11]
```diff
@@ -0,0 +1,280 @@
...
+
+* [Feature]: Address all the package installation rules for package namespaces work in PackageReference and PackageDownload scenarios. - [#11035](https://github.com/NuGet/Home/issues/11035)
+
+* [Feature]: Address all the package installation rules for package namespaces work in Packages Config style projects - [#11036](https://github.com/NuGet/Home/issues/11036)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* Consider removing the extensibility that allows other package managers to be shown as compatible - [#6623](https://github.com/NuGet/Home/issues/6623)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:65]: issue title needs wordsmithing.

File: docs/release-notes/NuGet-6.0.md [hunk 12]
```diff
@@ -0,0 +1,280 @@
...
+
+* Remove obsolete core APIs - [#10940](https://github.com/NuGet/Home/issues/10940)
+
+* Merge NuGet.SolutionRestoreManager.Interop into NuGet.VisualStudio - [#10957](https://github.com/NuGet/Home/issues/10957)
+
+* Remove RuntimeEnvironmentHelper.IsDev14 - [#11000](https://github.com/NuGet/Home/issues/11000)
+
+* Update Newtonsoft.Json to 13.0.1 - [#11095](https://github.com/NuGet/Home/issues/11095)
+
+* [DCR]: Rename packageNamespaces feature to packageSourceMapping - [#11205](https://github.com/NuGet/Home/issues/11205)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:79]: Remove this, it was a change within the original release, so as far as customers are concerned, they never knew about package namespaces to begin with.

**Comment by nkolev92** [comment]
This issue: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2496 should be a part of this PR as well (or at least a part of the documentation). 

@zivkan do you agree?

File: docs/release-notes/NuGet-6.0.md [hunk 13]
```diff
@@ -0,0 +1,387 @@
...
+  &lt;packageSource key="nuget.org"&gt;
+    &lt;package pattern="*" /&gt;
+  &lt;/packageSource&gt;
+  &lt;packageSource key="contoso.com"&gt;
+    &lt;package pattern="Contoso.*" /&gt;
+  &lt;/packageSource&gt;
+&lt;/packageSourceMapping&gt;
+</code></pre>
+
+You can read more about source mapping in our [blog released earlier this year](https://devblogs.microsoft.com/nuget/introducing-package-source-mapping/).
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.0.md:76]: Link the local docs instead/as well?

**Review by nkolev92 — COMMENTED** [review]
Love the new section, makes a lot of sense. 

There's a few things left in that are consequences of feature renaming, that I think we can                                                         cleanup. 

I'd be happy to do it, but also understand if we think it's an overkill and I can just approve the PR as is.

File: docs/release-notes/NuGet-6.0.md (hunk 13 above)
Comment (JonDouglas) [reply docs/release-notes/NuGet-6.0.md:76]: lol yes. I wrote these blurbs prior to merging your docs ;)

**Comment by JonDouglas** [comment]
> There's a few things left in that are consequences of feature renaming, that I think we can cleanup.

I have no strong feelings. Things change and the history is sometimes important. If we change anything, it should be cleaned up in the artifact as well such as the issue's title. Since we autogenerate notes, I think it's overkill this release, but would be willing to help clean these up too.

**Review by nkolev92 — APPROVED** [review]

**Comment by nkolev92** [comment]
> Since we autogenerate notes, I think it's overkill this release, but would be willing to help clean these up too.

We're certainly not lacking in other work we could be doing. 
So I'm with it as is.

**Pull request merged**